docs/docs/integrations/promotions/voucherify.md

```diff
@@ -0,0 +1,146 @@
+- `composable-ui/src/components/cart/cart-promotions.tsx`: Displays auto-applied cart promotions.
+
+## Integrating Voucherify with Composable UI
+
+### First steps
+
+1. [Create a Voucherify account](https://app.voucherify.io/#/signup).
+2. In Voucherify Dashboard, [set Discount Application Rule to "Partial"](https://support.voucherify.io/article/604-stacking-rules#application-rules)
+   Please go to:
+- `Redemptions` tab on the left side of the Voucherify dashboard
+- Select `Stacking rules`
+- In `Discounts' application rules` select `Edit settings`
+- Change the `Application rule` to `Partial - inapplicable (not met criteria) discounts are omitted`
+3. Retrieve your API keys from your Voucherify dashboard and set the following environment variables in `composable-ui/.env` and `scripts/.env` files:
+
+- Go to `Project Settings` in the Voucherify dashboard
+   - Click on your name in the upper right corner.
+   - From the list, select `Project Settings`.
+   - Find the section with `Application Keys`.
+   - Copy or make a note of the `Application ID` and `Secret Key`.
+   - Make a note of your [API endpoint](https://docs.voucherify.io/docs/api-endpoints).
+
+     | Shared Cluster   | Endpoint                        |
+     |------------------|---------------------------------|
+     | Europe (default) | `https://api.voucherify.io`     |
+     | United States    | `https://us1.api.voucherify.io` |
+     | Asia (Singapore) | `https://as1.api.voucherify.io` |
+
+Fill in the following variables with the noted values. The following three environment variables should be in both `composable-ui/.env` and `scripts/.env`.
+```code
+VOUCHERIFY_API_URL=https://api.voucherify.io
+VOUCHERIFY_APPLICATION_ID=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxxx
+VOUCHERIFY_SECRET_KEY=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxxx
+```
+
+**Caution:** Ensure you never expose your Voucherify API keys in the NEXT_PUBLIC_* environment variables or client-side code. Take the necessary steps to ensure that secret keys are never disclosed to the public.
+
+
+### Populating the Products Using the Script (optional)
+
+Running a script that adds products to Voucherify is optional. Without running it, you will still be able to perform validation and redemption.
+However, it may be required in the future if other types of discounts are implemented.
+
+**Note:** If you are using the free plan and sandbox project, keep in mind that it has a limit of 100 API calls per hour. 
+With the basic plan you should be able to perform the `voucherify-preconfigure` action without any problems. 
+However, on subsequent attempts to add products, you may get an error: `429 - TOO_MANY_REQUESTS`.
+In this situation, you will only be able to make further attempts in the next hour. You can also use the app without adding products.
+
+
+> **_NOTE:_**  If you are using the Free plan and Sandbox project, keep in mind that it has a limit of 100 API calls per hour.
```

Comment:
@weronika-kurczyna this NOTE looks to be a duplicate of the Note that starts on line 100. May want to remove one of these.

![image](https://github.com/composable-com/composable-ui/assets/13950845/69104562-4357-4cb5-a83c-e3486264ce8f)

Reply:
Ah, sorry for the confusion. It has been corrected.

Reply:
Thank you!